src/services/dashboards/all-dashboards/modules/DashboardBoardList.vue

```diff
@@ -147,13 +148,16 @@ export default defineComponent<DashboardBoardListProps>({
             dashboardTotalCount: computed<number>(() => props.dashboardList.length ?? 0),
             dashboardScopeKey: computed(() => (props.scopeType === DASHBOARD_SCOPE.DOMAIN ? DOMAIN_SCOPE_KEY : PROJECT_SCOPE_KEY)),
             projectItems: computed(() => store.getters['reference/projectItems']),
-            hasManagePermission: useManagePermissionState(),
+            hasManagePermission: computed(() => {
+                const routeName = props.scopeType === 'domain' ? MENU_ID.DASHBOARDS_WORKSPACE : MENU_ID.DASHBOARDS_PROJECT;
```

Comment:
I would recommend you there is a constant 
```suggestion
                const routeName = props.scopeType === DASHBOARD_SCOPE.DOMAIN ? MENU_ID.DASHBOARDS_WORKSPACE : MENU_ID.DASHBOARDS_PROJECT;
```

Reply:
🫡